code/datums/ammo/bullet/sniper.dm

```diff
@@ -135,7 +135,7 @@
 				if(amr.focused_fire_counter < 2) // Can stack up to twice.
 					amr.focused_fire_counter += 1
 				else
-					amr.focused_fire_counter = 2
+					amr.focused_fire_counter = -1
```

Comment:
This should be 0 else you won't get the one hit message; or you need to change the other handling of this counter. Just changing it to 0 though doesn't fix it incorrectly having the blue laser for the first hit. So do check that implementation too & test.

Reply:
yea I didn't realize the laser and counter were different things for some reason